types/trusted-types/lib/index.d.ts

```diff
@@ -0,0 +1,64 @@
+// The main type definitions. Packages that do not want to pollute the global
```

Comment:
Not sure if it is possible, but can this be recorded as a branch from trusted-types/index.d.ts ?

Reply:
From what I've read git doesn't explicitly track file renaming, and instead uses heuristics to detect removal/addition of similar files. This file just seems to be too different from the original for git to realize.